Mage.Tests/src/test/java/org/mage/test/cards/abilities/oneshot/exile/BanishmentTest.java

```diff
@@ -0,0 +1,57 @@
+package org.mage.test.cards.abilities.oneshot.exile;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestCommander4Players;
+
+public class BanishmentTest extends CardTestCommander4Players {
+
+    @Test
+    public void testBanishment() {
+        addCard(Zone.BATTLEFIELD, playerA, "Plains", 4);
```

Comment:
Same with the comment above.
PlayerA should also get a Memnite to make sure you aren't exiling permanents you control.
The filter you set for TargetPermanent will stop you from targeting your own memnite as the initial permanent to exile (you will target one of the ones an opponent within your range of influence controls).

Reply:
done